src/plugins/analysis/file_system_metadata/docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM fkiecad/fact_extractor:latest
```

Comment:
I guess you use the extractor as it supposedly has additional filesystem support?

Reply:
exactly. No point in setting up another container when the extractor already supports mounting all of them for extraction purposes.